test/integration_test.exs

```diff
@@ -85,35 +96,18 @@ defmodule Janus.Plugin.VideoRoom.IntegrationTest do
 
   test "Publish without join", %{session: session} do
     assert {:ok, pub_handle} = VideoRoom.new_handle(session)
-    properties = %VideoRoom.CreateRoomProperties{}
-
-    VideoRoom.destroy(session, @test_room_id, pub_handle)
-
-    assert {:ok, @test_room_id} =
-             VideoRoom.create(session, @test_room_id, properties, pub_handle, @plugin_admin_key)
-
     publisher_config = %VideoRoom.PublisherConfig{display_name: "pub1"}
 
     # premature publish
     assert {:error, {:janus_videoroom_error_join_first, 424, _description}} =
              VideoRoom.publish(session, publisher_config, pub_handle, TestFixtures.sdp_offer())
+
+    assert :ok = VideoRoom.destroy_handle(session, pub_handle)
   end
 
   @tag :focus
```

Comment:
What does `:focus` tag do? Seems not to be part of known exunit tags.

Reply:
Testing leftover, I've used `mix test --only focus`